internal/pkg/service/buffer/store/schema/schema_test.go

```diff
@@ -89,129 +91,89 @@ func TestSchema(t *testing.T) {
 			"file/",
 		},
 		{
-			s.Files().Opened().Prefix(),
-			"file/opened/",
+			s.Files().InState(filestate.Opened).InReceiver(receiverKey).Prefix(),
+			"file/opened/00000123/my-receiver/",
 		},
 		{
-			s.Files().Opened().InExport(exportKey).Prefix(),
+			s.Files().InState(filestate.Opened).InExport(exportKey).Prefix(),
 			"file/opened/00000123/my-receiver/my-export/",
 		},
 		{
 			s.Files().Opened().ByKey(fileKey).Key(),
 			"file/opened/00000123/my-receiver/my-export/2006-01-02T08:04:05.000Z",
 		},
 		{
-			s.Files().Closing().Prefix(),
-			"file/closing/",
-		},
-		{
-			s.Files().Closing().InExport(exportKey).Prefix(),
-			"file/closing/00000123/my-receiver/my-export/",
+			s.Files().Opened().Prefix(),
+			"file/opened/",
 		},
 		{
-			s.Files().Closing().ByKey(fileKey).Key(),
-			"file/closing/00000123/my-receiver/my-export/2006-01-02T08:04:05.000Z",
+			s.Files().Closing().Prefix(),
+			"file/closing/",
 		},
 		{
 			s.Files().Closed().Prefix(),
 			"file/closed/",
 		},
-		{
-			s.Files().Closed().InExport(exportKey).Prefix(),
-			"file/closed/00000123/my-receiver/my-export/",
-		},
-		{
-			s.Files().Closed().ByKey(fileKey).Key(),
-			"file/closed/00000123/my-receiver/my-export/2006-01-02T08:04:05.000Z",
-		},
-		{
-			s.Files().Importing().Prefix(),
-			"file/importing/",
-		},
-		{
```

Comment:
Zbytocne tu boli testy pre kazdy stav, staci jeden test, kedze je to iba skratka pre `.InState(...)`